### PR TITLE
Add exception handling for breadcrumb function

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR add exception handling for missing document when query for breadcrumb.

### CODE REVIEW

File: src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php [hunk 1]
```diff
@@ -142,11 +143,11 @@ public function breadcrumbFunction($uuid)
         $webspaceKey = $this->requestAnalyzer->getWebspace()->getKey();
         $locale = $this->requestAnalyzer->getCurrentLocalization()->getLocale();
 
-        return $this->navigationMapper->getBreadcrumb(
-            $uuid,
-            $webspaceKey,
-            $locale
-        );
+        try {
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php]: should this not also be added to all the other functions basing on the uuid

**Comment by wachterjohannes** [comment]
@alexander-schranz fixed

**Comment by alexander-schranz** [comment]
@wachterjohannes tests failing

**Comment by wachterjohannes** [comment]
@alexander-schranz should go green now :)

File: src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineJoinDescriptor.php [hunk 2]
```diff
@@ -64,11 +64,11 @@ class DoctrineJoinDescriptor
     private $joinMethod;
 
     public function __construct(
-        string $entityName,
```
Comment (wachterjohannes) [inline src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineJoinDescriptor.php]: @alexander-schranz @danrot was this a merge issue from another PR?

**Pull request merged**